packages/serverless-contracts/package.json

```diff
@@ -19,10 +19,12 @@
   "files": [
     "dist"
   ],
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.esm.js",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
   "types": "dist/types/index.d.ts",
   "scripts": {
+    "bench": "pnpm tsx src/__benches__/index.ts",
```

Comment:
Maybe this folder should be outside of `src`?
We need to make sure this is not included in the code shipped to npm

Reply:
Since nothing from this folder is exported from `src/index.ts`, there will not be any `js` code exported. And for types the `src/__benches__` folder is excluded from the `tsconfig.build.json`.

I wanted to keep the benches in the `src` folder because, in the long run, our benches will be done through `vitest bench` (once CodSpeed supports it), and our benches will live right next to the source code as tests do.

WDYT?

Reply:
I see, and in this case `src/__benches__/index.ts` will disappear right ? In order to minimize the rework when that is done, we can put it in `__benches__` and directly import from all the benches subdirectories (skip all the `index.ts` exports). WDYT?

Reply:
Yes you are completely right, I will do that 😉